service/rtc/call.go

```diff
@@ -63,9 +64,25 @@ func (c *call) addSession(cfg SessionConfig, rtcConn *webrtc.PeerConnection, clo
 	}
 
 	c.sessions[cfg.SessionID] = s
+
+	c.sendCodecSupportMap(log)
```

Comment:
This is where we send the updated map after a new session joins. It's important that session also gets it, so this call must happen after updating the sessions map (line above).